taworks/taform/views.py

```diff
@@ -23,7 +23,6 @@
 from threading import Thread
 import pandas as pd
 from django.db.models import Count, Case, When, IntegerField, Avg
-import datetime
```

Comment:
I use datetime for some of my code - do we need to remove it?

Reply:
it was already imported further above.